<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CDbase</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lblSearchTitle">
            <property name="text">
             <string>CDTitle:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSearchTitle"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lblSearchAuthor">
            <property name="text">
             <string>CDAuthor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSearchAuthor"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="lblSearchYear">
            <property name="text">
             <string>ReleaseYear:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSearchYear"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="lblSearchGenre">
            <property name="text">
             <string>Genre:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSearchGenre"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="btnResetSearch">
            <property name="text">
             <string>CancelFilter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSearch">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tvDisplayedCDs">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QGraphicsView" name="gvBooklet"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectPicture">
            <property name="text">
             <string>btnSelectPicture</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="lblCDName">
              <property name="text">
               <string>CDTitle:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leCDName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="lblCDAuthor">
              <property name="text">
               <string>CDAuthor:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leCDAuthor"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lblCDYear">
              <property name="text">
               <string>ReleaseYear:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leCDYear"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="lblAddGenre">
              <property name="text">
               <string>Genre:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leGenre"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="l_9_song_edit">
        <item>
         <layout class="QHBoxLayout" name="hl_5_5_CD_song">
          <item>
           <widget class="QLabel" name="lblSong">
            <property name="text">
             <string>Song:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSongTitle"/>
          </item>
          <item>
           <widget class="QTimeEdit" name="teSongDuration">
            <property name="currentSection">
             <enum>QDateTimeEdit::MinuteSection</enum>
            </property>
            <property name="displayFormat">
             <string>mm:ss</string>
            </property>
            <property name="currentSectionIndex">
             <number>0</number>
            </property>
            <property name="timeSpec">
             <enum>Qt::UTC</enum>
            </property>
            <property name="time">
             <time>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
             </time>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="l_5_6_song_buttons">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnAddSong">
            <property name="text">
             <string>btnAdd</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditSong">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>btnEdit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveSong">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>btnRemove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tvDisplayedSongs">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="l_9_bottom_buttons">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>btnCancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddCD">
          <property name="text">
           <string>AddCD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSaveEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>EditCD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemoveCD">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>RemoveCD</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuUkoncit">
   <property name="text">
    <string>btnExit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tvDisplayedCDs</tabstop>
  <tabstop>leCDName</tabstop>
  <tabstop>leCDAuthor</tabstop>
  <tabstop>leCDYear</tabstop>
  <tabstop>leGenre</tabstop>
  <tabstop>btnSelectPicture</tabstop>
  <tabstop>gvBooklet</tabstop>
  <tabstop>leSongTitle</tabstop>
  <tabstop>teSongDuration</tabstop>
  <tabstop>btnAddSong</tabstop>
  <tabstop>btnEditSong</tabstop>
  <tabstop>btnRemoveSong</tabstop>
  <tabstop>tvDisplayedSongs</tabstop>
  <tabstop>btnAddCD</tabstop>
  <tabstop>btnSaveEdit</tabstop>
  <tabstop>btnRemoveCD</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>leSearchTitle</tabstop>
  <tabstop>leSearchAuthor</tabstop>
  <tabstop>leSearchYear</tabstop>
  <tabstop>leSearchGenre</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>btnResetSearch</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>addNewCD()</slot>
 </slots>
</ui>
